docs/modules/ROOT/pages/component-kserve-explainer.adoc

```diff
@@ -0,0 +1,29 @@
+= KServe explainer
+
+image::trustyai-kserve-explainer.svg[TrustyAI KServe architecture diagram]
+
+The TrustyAI KServe Explainer is a component that provides explanations for predictions made by using the built-in KServe explainer support footnote:fn-kserveexplainer[Documentation available at https://kserve.github.io/website/0.12/modelserving/explainer/explainer/[KServe explainers section].]. It supports xref:local-explainers.adoc#lime[LIME] and xref:local-explainers.adoc#shap[SHAP] explanation methods, configurable directly within KServe `InferenceServices`.
+
+== Features
+
+- **Explainability**: Integrated support for xref:local-explainers.adoc#lime[LIME] and xref:local-explainers.adoc#shap[SHAP] explanation methods to interpret model predictions via the `:explain` endpoint.
+
+== Deployment on KServe
+
+The TrustyAI explainer can be added to KServe `InferenceServices` and can be configured to use either xref:local-explainers.adoc#lime[LIME] or xref:local-explainers.adoc#shap[SHAP] explanation methods by modifying the YAML configuration.
+
+When deployed, Istio manages the routing of requests to the appropriate container. Calls to `/v1/models/model:predict` will be sent to the predictor container, while calls to `/v1/models/model:explain` will be sent to the explainer container. The payloads for both endpoints are the same, but the `:predict` endpoint returns the model's prediction, while the `:explain` endpoint returns an explanation of the prediction.
```

Comment:
I would remove all Istio mentions from the document: from user PoV it is KServe that does it, Istio is an implementation detail. Moreover it is not the only option that is available ([Kourier should work too](https://kserve.github.io/website/latest/admin/serverless/kourier_networking/)).
Same for Istio logo: use KServe logo directly so we don't expose too many details

Reply:
Replaced Istio with KServe